codefresh/apk.yml

```diff
@@ -28,6 +28,9 @@ steps:
 
   docker_build:
     type: "parallel"
+    title: "Build Docker Images"
+    description: "Build Docker images for all recent alpine releases"
+    stage: Prepare
```

Comment:
is this stage `Prepare` or `Build`?

Reply:
All things relative. We are preparing the docker images that will be used to build the alpine packages. The docker images are not the deliverable, the packages are. So building the images is just a preparatory step.

Reply:
![image](https://user-images.githubusercontent.com/52489/58594795-8cd8de80-8223-11e9-8258-abbe96ed203c.png)